### PR TITLE
GraphQL configuration for UI

### PR DESCRIPTION
**Pull request by peoplenarthax**
Add minimum changes in the server and client to allow mutations and queries from the front end. Also ugly form for add boardgames is there

### CODE REVIEW

File: packages/ui/public/index.html [hunk 1]
```diff
@@ -16,42 +16,9 @@
...
 
     <style>
-      html, body, #root {
-        position: relative;
-        height: 100%;
-        width: 100%;
-        margin: 0;
-        padding: 0;
-
-        font-family: 'Nunito', sans-serif;
```
Comment (beeequeue) [inline packages/ui/public/index.html:26]: lets put these back

File: packages/ui/schema.graphql [hunk 2]
```diff
@@ -0,0 +1,81 @@
+type Boardgame {
```
Comment (beeequeue) [inline packages/ui/schema.graphql]: this file should be ignored and not committed

File: packages/ui/src/pages/boardgame/add.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
+import React, { useState, useCallback } from 'react'
+import { hot } from 'react-hot-loader'
+import AceEditor from 'react-ace'
+import Slider from 'rc-slider'
+import styled from 'styled-components'
+import { useMutation } from '@apollo/react-hooks'
+
+
+import 'ace-builds/src-noconflict/mode-json'
```
Comment (beeequeue) [inline packages/ui/src/pages/boardgame/add.tsx:8]: oof

Comment (peoplenarthax) [reply packages/ui/src/pages/boardgame/add.tsx:8]: sorry

File: packages/ui/package.json [hunk 4]
```diff
@@ -14,12 +14,17 @@
...
     "graphql": "14.5.8",
     "graphql-tag": "2.10.1",
+    "rc-slider": "8.7.1",
     "react": "16.11.0",
+    "react-ace": "8.0.0",
     "react-dom": "16.11.0",
     "react-hot-loader": "4.12.15",
     "react-router-dom": "5.1.2",
+    "style-loader": "1.0.0",
+    "styled-components": "^4.4.1",
```
Comment (beeequeue) [inline packages/ui/package.json]:
```suggestion
    "styled-components": "5.0.0-rc.1",
```

File: packages/ui/src/pages/boardgame/rc-slider-styles.ts [hunk 5]
```diff
@@ -0,0 +1,261 @@
...
+  border-style: solid;
+}
+.rc-slider-tooltip-placement-top .rc-slider-tooltip-arrow {
+  bottom: 4px;
+  left: 50%;
+  margin-left: -4px;
+  border-width: 4px 4px 0;
+  border-top-color: #6c6c6c;
+}
+`
```
Comment (beeequeue) [inline packages/ui/src/pages/boardgame/rc-slider-styles.ts:261]: can we try to do the raw-loader trick i talked about earlier? maybe not in this PR but it'd be nice to not have this code in our codebase

Comment (peoplenarthax) [reply packages/ui/src/pages/boardgame/rc-slider-styles.ts:261]: We will remove this code within 1 or 2 PRs regarding forms, so I wouldnt worry much

**Review by beeequeue — APPROVED** [review]

**Pull request merged**